src/utilities/helpers.ts

```diff
@@ -138,9 +138,9 @@ export async function fetchAllResources<T = Object>(endpoint: (params: Object) =
         allTotal = total
       }
 
-      if (total !== allTotal) {
-        throw new Error('Mismatch between "total" values between requests')
-      }
+      // if (total !== allTotal) {
+      //   throw new Error('Mismatch between "total" values between requests')
+      // }
```

Comment:
I wanted to double check this before complete removal.

This seems to check that the `total` stays the same throughout all pages of a paged request, and if it doesn't then an error is thrown.

This feels to me like it is actually expected behaviour that the total could change between requesting page one and page two of a request. Erroring if the total changes is probably not quite what we want to do.

I came across this as during testing. As our mocks no longer have state, it is quite possible that the total is different between requesting page one and page two (etc etc).

All in all I think deleting this is safe, and desirable, but thought I'd ask for a second opinion incase.

I'll be either deleting or uncommenting this before merge (probably deleting)